pkg/configuration/router.go

```diff
@@ -62,3 +100,114 @@ func updateOrganizationConfig(logger log.Logger, repo Repository) http.HandlerFu
 		json.NewEncoder(w).Encode(cfg)
 	}
 }
+
+func getOrganizationLogo(logger log.Logger, repo Repository, bucketFactory storage.BucketFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		requestID := moovhttp.GetRequestID(r)
+		organization := route.GetOrganization(w, r)
+		if organization == "" {
+			logger.Log("configuration", "get organization logo called with no organization header", "requestID", requestID)
+			return
+		}
+
+		bucket, err := bucketFactory()
+		if err != nil {
+			logger.Log("configuration", "problem retrieving logo image", "error", err, "organization", organization, "requestID", requestID)
+			moovhttp.Problem(w, err)
+			return
+		}
+		defer bucket.Close()
+
+		rdr, err := bucket.NewReader(r.Context(), makeDocumentKey(organization), nil)
+		if err != nil {
+			msg := "error retrieving logo file"
+			if gcerrors.Code(err) == gcerrors.NotFound {
```

Comment:
Is this a GCP specific error? What about other supported platforms?

Reply:
My understanding is these are applicable to all of our bucket types. The unit test around it works for our file bucket. From the docs:
```// Package gcerrors provides support for getting error codes from
// errors returned by Go CDK APIs.```

Reply:
A `blob.Reader` (what we get back from `bucket.NewReader`) implements `io.Reader`, so can we use `io.EOF` or `os.IsNotExist` by chance? No idea if those two work - maybe there's a github issue talking about this? 

https://github.com/google/go-cloud/blob/v0.20.0/blob/blob.go#L105-L110

Reply:
Oh, `gcerrors` threw me for a loop. If that's their package to wrap and works for all providers I'm good with it.

Reply:
Sweet. I think we're good on this then.